llama_stack/cli/stack/_build.py

```diff
@@ -71,6 +71,12 @@ def run_stack_build_command(args: argparse.Namespace) -> None:
     else:
         image_name = args.image_name
 
+    if not image_name:
+        image_name = args.image_type
+        print(
+            f"No image name provided and could not detect one from environment. Using image_name: llamastack-{image_name}"
```

Comment:
The log message and the code behave differently - while the code sets args.image_type to image_name, the log message includes a “llamastack-” prefix. Isn’t that a bit confusing?